dw-select-dialog-input.js

```diff
@@ -67,6 +73,24 @@ export class DwSelectDialogInput extends LitElement {
           color: var(--mdc-theme-text-hint-on-surface, rgba(0, 0, 0, 0.38));
           ${TypographyLiterals.subtitle1};
         }
+
+        .new-tag {
+          background-color: #259b24;
+          color: #ffffff;
+          display: inline-flex;
+          ${unsafeCSS(TypographyLiterals.caption)};
+          padding: 0px 8px;
```

Comment:
Never set any padding `0px` if padding is not required. Set right-left padding if only right and left padding are required. 